webknossos/webknossos/dataset/view.py

```diff
@@ -299,9 +241,8 @@ def _get_mag1_bbox(
     def write(
         self,
         data: np.ndarray,
-        offset: Optional[Vec3IntLike] = None,  # deprecated, relative, in current mag
-        json_update_allowed: bool = True,
```

Comment:
why doesn't the `View.write` method have a `allow_resize` parameter (only `MagView.write` has it currently)?

Reply:
See my comment above. A `View` doesn't have a reference to its `Layer`, so it cannot resize the `Layer`.